docs/articles/documentation/recipes/import-third-party-modules.md

```diff
@@ -33,3 +33,5 @@ test(`My Test`, async t => {
     // ...
 });
 ```
+
+[Full Example Code](https://github.com/DevExpress/testcafe-examples/tree/master/examples/import-third-party-modules)
```

Comment:
Move the link to the 'testcafe-examples'  before the code (as for other recipes).